tests/unit/models/test_projects.py

```diff
@@ -40,6 +40,8 @@
 
 # required project data for core24 snaps
 CORE24_DATA = {"base": "core24", "grade": "devel"}
+VALID_DURATIONS = ["10ns", "10us", "10ms", "10s", "10m", "10m4s3us"]
```

Comment:
I added a separate data entry for a valid duration here, as `"10m4s3us"` is a valid duration string. Snapd uses [time.ParseDuration](https://pkg.go.dev/time#ParseDuration) to parse this, and the resulting service file contains a value something like `TimeoutStartSec=604.000003`.